CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ find_package(LLVM CONFIG REQUIRED)
 find_package(Swift CONFIG REQUIRED)
 find_package(Clang CONFIG REQUIRED)
 
-add_executable(swift-repl main.cpp REPL.cpp)
+add_executable(swift-repl main.cpp REPL.cpp JIT.cpp)
 target_include_directories(swift-repl PRIVATE ${SWIFT_INCLUDE_DIRS} ${LLVM_INCLUDE_DIRS} ${CLANG_INCLUDE_DIRS})
+target_link_libraries(swift-repl PRIVATE LLVMExecutionEngine LLVMOrcJIT)
```

Comment:
Fold this into the line below. You can also reformat it so you don't have one massive line.